core/plugins/org.polarsys.capella.core.transition.system/src/org/polarsys/capella/core/transition/system/handlers/traceability/ReconciliationTraceabilityHandler.java

```diff
@@ -81,8 +84,32 @@ protected void initializeMapping(EObject source, EObject target, IContext contex
 
   }
 
+  /**
+   * Map the Data package and its Predefined Types package if possible
+   * @param source
+   * @param target
+   * @param context
+   * @param map
+   */
   protected void initializeDataPkg(DataPkg source, DataPkg target, IContext context, LevelMappingTraceability map) {
-    initializeDataType(source, target, context, map);
+    BlockArchitecture sourceBlockArchitecture = BlockArchitectureExt.getRootBlockArchitecture(source);
+    BlockArchitecture targetBlockArchitecture = BlockArchitectureExt.getRootBlockArchitecture(target);
+    if (sourceBlockArchitecture != null && targetBlockArchitecture != null) {
+      DataPkg sourceDataPkg = BlockArchitectureExt.getDataPkg(sourceBlockArchitecture);
+      DataPkg targetDataPkg = BlockArchitectureExt.getDataPkg(targetBlockArchitecture);
+      if (source == sourceDataPkg && target == targetDataPkg) {
+        addMapping(map, sourceDataPkg, targetDataPkg, context);
+        Optional<DataPkg> sourcePredefinedTypes = sourceDataPkg.getOwnedDataPkgs().stream()
+            .filter(pkg -> pkg.getName().equals(NamingConstants.PredefinedTypesCmd_predefinedDataTypePkg_name))
```

Comment:
NamingConstants.PredefinedTypesCmd_predefinedDataTypePkg_name.equals to avoid NPE